packages/hash/frontend/src/testUtils/MockTheme.tsx

```diff
@@ -0,0 +1,14 @@
+import { createTheme, ThemeProvider } from "@mui/material";
```

Comment:
I've created a Mock Theme Provider for Jest to make the `jest` tests succeed.

Reply:
Moving the test fixes to a different PR now: https://github.com/hashintel/hash/pull/378